diff --git i/webform_import.module w/webform_import.module
index 3268cb2..802b36d 100644
--- i/webform_import.module
+++ w/webform_import.module
@@ -79,7 +79,7 @@ function webform_import_menu() {
  */
 function webform_import_csvtemplate($node, $type) {
   $types = _webform_import_field_key_options();
-  $filename = check_plain($node->title) . '_upload.csv';
+  $filename = str_replace(' ', '_', check_plain($node->title)) . '_upload.csv';
   $headers = array();
   $node->webform['components']['-1'] = array(
     'name' => 'Submission ID',
